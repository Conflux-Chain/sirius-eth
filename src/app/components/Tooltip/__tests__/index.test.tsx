import React from 'react';
import { render } from '@testing-library/react';
import { Tooltip } from '..';

describe('<Tooltip  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <Tooltip text="test text">test content</Tooltip>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
