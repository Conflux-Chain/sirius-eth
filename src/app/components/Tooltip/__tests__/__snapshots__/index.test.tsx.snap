// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tooltip  /> should match snapshot 1`] = `
.c0 .tooltip-content.sirius-tooltip {
  font-variant-numeric: initial;
  text-transform: initial;
  border-radius: 0;
  padding: 0.2857rem 0.7143rem 0.3571rem;
  font-weight: 400;
  font-size: 12px;
  color: #cdcdcd;
  line-height: 1.1429rem;
  min-width: 40px;
  max-width: 40rem;
}

.c0 a {
  color: #00acff;
}

.c0 a:hover {
  color: #3dc0ff;
}

@media (max-width:768px) {
  .c0 .tooltip-content.sirius-tooltip {
    max-width: 24rem;
    white-space: normal;
    word-break: break-all;
  }
}

<span
  class="c0"
>
  <div
    class="jsx-726172838 tooltip-wrapper"
  >
    <div
      class="jsx-726172838 tooltip "
    >
      test content
    </div>
  </div>
</span>
`;
